apps/trackers/jira/constants.py

```diff
@@ -19,3 +21,11 @@
     "target_release": "customfield_12311240",
     "target_version": "customfield_12319940",
 }
+
+
+class TrackersAppSettings(BaseSettings):
```

Comment:
I like the usage of the `pydantic_settings`, maybe we could standardize on that.

Reply:
yeah, it has lots of benefits, I'll bring up a thread in slack